classes/fields/color.php

```diff
@@ -111,7 +111,7 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 					// @todo Ask for a specific format in error message
 					$errors[] = __( 'Invalid value provided for this field.', 'pods' );
 				}
-			} elseif ( ! in_array( strlen( $color ), array( 3, 6 ), true ) ) {
+			} elseif ( $color && ! in_array( strlen( $color ), array( 3, 6 ), true ) ) {
```

Comment:
Not sure it'll make any difference in this context, but lets change it to `! empty( $color )` to match the logic before the breakage

Reply:
Won't matter since it's already declared but fine with me. Also enhances readability so that's a plus :)